labs/coding-101-rest-basics-ga/3.md

```diff
@@ -73,7 +73,7 @@ Study the Ticket API in APIC-EM:
 
   ![](/posts/files/coding-101-rest-basics-ga/assets/images/refguide4.png)
 
-* In the Parameters section, click **Model Schema**. Double-clicking the box moves the parameters to an editable box to try out the function.
+* In the Parameters section, click **Example Value**. Double-clicking the box moves the parameters to an editable box on the left to try out the function.
 
```

Comment:
Leaving this PR open so we can amend it with a screenshot that matches the text.